index.js

```diff
@@ -23,6 +23,80 @@ server.get("/api/projects", (req, res) => {
     .catch(() => res.status(500).json({ message: "Server Error" }));
 });
 
+server.post("/api/projects", (req, res) => {
+  const { name, description } = req.body;
+
+  if (!name || !description) {
+    res.status(422).json({ message: "Name and Description required!" });
+  } else {
+    db("projects")
+      .insert({ name, description })
+      .then(result => {
+        if (result.rowCount) {
+          db("projects").then(projects => {
+            res.json(projects);
```

Comment:
For readability just try to stay as consistent as you can with the style you choose for chaining your `then` calls.